tests/conftest.py

```diff
@@ -77,6 +80,24 @@ def stubbed_sender() -> Keypair:
     return Keypair.from_seed(bytes([8] * PublicKey.LENGTH))
 
 
+@pytest.fixture(scope="session")
+def stubbed_sender_http() -> Keypair:
+    """Arbitrary known account to be used as sender."""
+    return Keypair.from_seed(bytes([10] * PublicKey.LENGTH))
+
+
+@pytest.fixture(scope="session")
+def stubbed_sender_memo() -> Keypair:
+    """Arbitrary known account to be used as sender."""
+    return Keypair.from_seed(bytes([11] * PublicKey.LENGTH))
+
+
+@pytest.fixture(scope="session")
+def stubbed_sender_token() -> Keypair:
+    """Arbitrary known account to be used as sender."""
+    return Keypair.from_seed(bytes([12] * PublicKey.LENGTH))
```

Comment:
Why do we need new stubbed senders? Can we reuse the existing ones?

Reply:
this is just to avoid figuring out the assert value blackhole.

Reply:
Yes. Please revert these extra stub values.